pkg/clusteragent/custommetrics/store.go

```diff
@@ -107,7 +121,23 @@ func (c *configMapStore) SetExternalMetricValues(added []ExternalMetricValue) er
 		}
 		c.cm.Data[key] = string(toStore)
 	}
-	return c.updateConfigMap()
+	if err := c.updateConfigMap(); err != nil {
```

Comment:
We could have metrics that relate to the rate of errors when accessing resources like CM/Nodes/Edpts ... ?
For later, but just to have your thoughts.